src/quo2/components/record_audio/record_audio/view.cljs

```diff
@@ -211,13 +211,43 @@
            (fn []
              (audio/destroy-recorder @recorder-ref)
              (reset! recorder-ref nil))
+           recorder-on-meter
+           (fn []
+             (let [current-timestamp  (.now js/Date)
```

Comment:
Prefer to use `(utils.datetime/timestamp)` if possible. It should work without modification to do math with ms precision. For example https://github.com/status-im/status-mobile/blob/de86586208aa61577aa9f22665e849d574e79a1c/src/status_im/ui/screens/communities/community.cljs#L32